js/src/components/enhanced-conversion-tracking-settings/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useAcceptedCustomerDataTerms from '.~/hooks/useAcceptedCustomerDataTerms';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+import Section from '.~/wcdl/section';
+import PendingNotice from '.~/components/enhanced-conversion-tracking-settings/pending-notice';
+import Toggle from './toggle';
+
+const DESCRIPTION = (
+	<p>
+		{ __(
+			'Improve your conversion tracking accuracy and unlock more powerful bidding. This feature works alongside your existing conversion tags, sending secure, privacy-friendly conversion data from your website to Google.',
+			'google-listings-and-ads'
+		) }
+	</p>
+);
+
+const TITLE = __( 'Enhanced Conversion Tracking', 'google-listings-and-ads' );
+
+/**
+ * Renders the settings panel for enhanced conversion tracking
+ */
+const EnhancedConversionTrackingSettings = () => {
+	const { googleAdsAccount } = useGoogleAdsAccount();
+	const { acceptedCustomerDataTerms } = useAcceptedCustomerDataTerms();
+
+	if ( ! googleAdsAccount || ! googleAdsAccount.id ) {
+		return null;
+	}
+
+	return (
+		<Section title={ TITLE } description={ DESCRIPTION }>
+			<Section.Card>
+				<Section.Card.Body>
+					{ ! acceptedCustomerDataTerms && <PendingNotice /> }
```

Comment:
There is still an error after disconnecting Google Ads account.

![image](https://github.com/woocommerce/google-listings-and-ads/assets/17420811/82ad89dd-5be5-46bf-8b6e-01c88cb61deb)